### DISCUSSION TITLE
Gateway rule and properties for Inner rule

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3121] [Questions] [answered]
I have a business rule (implementing an algorithm) that needs to be run based on the value of a certain property (say Property1)
The main business rule needs several properties but the gateway rule would require only one property (Property1). If I am not mistaken I still have to pass all the properties (needed by main business rule) to the gateway rule ?

Kind Regards

**Comment by rockfordlhotka** [comment, marked as answer]
I believe that is correct.

**Reply by Chicagoan2016** [reply]
@rockfordlhotka Thank you Sir.
